pkgs/applications/graphics/drawio/default.nix

```diff
@@ -54,9 +54,15 @@ stdenv.mkDerivation rec {
   buildPhase = ''
     runHook preBuild
 
+  '' + lib.optionalString stdenv.isDarwin ''
+    cp -R ${electron}/Applications/Electron.app Electron.app
+    chmod -R u+w Electron.app
```

Comment:
Why is this required?

Reply:
`electron-builder` appears to build `dist/mac*/draw.io.app` directly on top of `Electron.app` by overwriting the files in the bundle. Not doing this results in an error when manipulating `Electron.app/Contents/Frameworks/Electron Helper.app/Contents/Info.plist`.